lapis/job_io/htcondor.py

```diff
@@ -31,36 +32,55 @@ def htcondor_job_reader(
         "DiskUsage_RAW": 1.024 / 1024 / 1024,
     },
 ):
-    htcondor_reader = csv.DictReader(iterable, delimiter=" ", quotechar="'")
-
-    for row in htcondor_reader:
-        if float(row[used_resource_name_mapping["walltime"]]) <= 0:
+    input_file_type = iterable.name.split(".")[-1]
```

Comment:
I would recommend to also make letters lower case to also match e.g. "JSON" or "CSV".

```suggestion
    input_file_type = iterable.name.split(".")[-1].lower()
```